ci/cron/src/Main.hs

```diff
@@ -262,7 +262,51 @@ docs = do
 
 check_releases :: String -> IO ()
 check_releases bash_lib = do
-    putStrLn $ "arg: " <> bash_lib
+    out <- shell $ unlines ["bash -c '",
```

Comment:
Seems like there is an extra level of shells here. If you are calling `bash`, you don’t need `shell`.

Reply:
I'm aware of the inefficiency here. `shell` is my own wrapper over `System.shell` (see lines 53 & 34), which happens to run `sh`, which would not run this script. I suppose it could be more efficient to just call `System.createProcess` here (and would allow me to pass in the current stdout directly, avoiding the buffering issue), but I don't think it's worth doing at this point.